NineChronicles.Headless.Executable/Program.cs

```diff
@@ -155,37 +169,31 @@ static void ConfigureSentryOptions(SentryOptions o)
             string? awsSecretKey = null,
             [Option(Description = "The AWS region for AWS CloudWatch (e.g., us-east-1, ap-northeast-2).")]
             string? awsRegion = null,
-            [Option(Description = "The lifetime of each transaction, which uses minute as its unit.  180 (m) by default.")]
+            [Option(Description =
+                "The lifetime of each transaction, which uses minute as its unit.")]
             int txLifeTime = 180,
-            [Option(Description = "The grace period for new messages, which uses second as its unit.  60 (s) by default.")]
+            [Option(Description =
+                "The grace period for new messages, which uses second as its unit.")]
             int messageTimeout = 60,
-            [Option(Description = "The grace period for tip update, which uses second as its unit.  60 (s) by default.")]
+            [Option(Description = "The grace period for tip update, which uses second as its unit.")]
             int tipTimeout = 60,
-            [Option(Description =
-                "A number that determines how far behind the demand the tip of the chain " +
-                "will publish `NodeException` to GraphQL subscriptions.  1150 blocks by default.")]
+            [Option(Description = "A number that determines how far behind the demand the tip of the chain " +
+                                  "will publish `NodeException` to GraphQL subscriptions.  1150 blocks by default.")]
             int demandBuffer = 1150,
             [Option("static-peer",
                 Description = "A list of peers that the node will continue to maintain.")]
             string[]? staticPeerStrings = null,
-            [Option("miner-count", Description = "The number of miner task(thread).")]
-            int minerCount = 1,
-            [Option(Description ="Run node without preloading.")]
+            [Option(Description = "Run node without preloading.")]
             bool skipPreload = false,
             [Option(Description = "Minimum number of peers to broadcast message.  10 by default.")]
             int minimumBroadcastTarget = 10,
-            [Option(Description =
-                "Number of the peers can be stored in each bucket.  16 by default.")]
+            [Option(Description = "Number of the peers can be stored in each bucket.  16 by default.")]
```

Comment:
If you have a opinion that the default value should be removed because Cocona shows it automatically, this default value also should be removed. Doesn't it?